pkgs/top-level/all-packages.nix

```diff
@@ -24195,6 +24195,10 @@ with pkgs;
 
   vcg = callPackage ../development/libraries/vcg { };
 
+  vencord = callPackage ../misc/vencord { };
+
+  vencord-web-extension = callPackage ../misc/vencord { buildWebExtension = true; };
```

Comment:
How about we move this to passthru?

Reply:
Not quite sure. These are really two different build modes at the end.

`buildWebExtension = true;` will only build the web extension (for webcord), while `buildWebExtension = false;` will only build the variant for Discord